README.md

```diff
@@ -347,28 +353,81 @@ Set the message size limit for all users. If set to zero, the size will be unlim
   - ignore => Ignore the failure of this test. Allow other tests to complete. Repeat this test the next time the client connects. This option is useful for testing and collecting statistics without blocking mail.
 
 
-##### REPORT_RECIPIENT
+## Reports
+
+##### PFLOGSUMM_TRIGGER
+
+  Enables a report being sent (created by pflogsumm) on a regular basis.
+  - **none** => No report
+  - daily_cron => Daily report for Yesterday will be sent
+  - logrotate => Full report for period will be sent when logs are rotated
```

Comment:
is this "full" different to the other options? Maybe specify that reporting will be weekly (have not checked the interval)

Reply:
It is different. Full means whatever was in the logs when they got rotated. If the log rotate period is monthly that will be a month, but there will also typically by a few entries for the new period. So if the period is daily, the report will often be for two days where one is very short. That is how it works now and what the cron alternative aims to fix.